### PR TITLE
An 325&329/terra console dbt tests

### PR DESCRIPTION
**Pull request by amasucci13**
Models & tests:
Daily Swap For LUNA/UST - New Users
LUNA Staked Supply - USD
Transactions per token
Total Transactions on Terra - New query, updated dbt model
Terra Fees by Denomination in UST
Daily Total Fees in USD
Terraswap Volume
Terraswap Top Swaps
Terraswap Top 5 Pools

### CODE REVIEW

File: tests/terra/consoles/consoles__terra_fees_UST__block_date-assert_no_gap.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{{ date_gaps(ref("consoles__terra_fees_UST"), [], "block_date") }}
+--Removed partition by currency since some of the smaller currencies have verified date gaps - AM 11/12
```
Comment (juls858) [inline tests/terra/consoles/consoles__terra_fees_UST__block_date-assert_no_gap.sql:2]: do we need this test if there gaps are ok?

**Review by juls858 — CHANGES_REQUESTED** [review]
do we need tests for date gaps when we know gaps are ok?

File: models/consoles/terra/consoles__terra_daily_total_fees_USD.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  SELECT 
+  block_timestamp::date as metric_date,
+  case when chain_id = 'columbus-5' then UPPER(SUBSTRING(fee[0]:amount[0]:denom::string, 2, 2)) 
+       else UPPER(SUBSTRING(fee[0]:denom::string, 2, 2)) end as fee_denom,
+  sum(case when chain_id = 'columbus-5' then fee[0]:amount[0]:amount
+       else fee[0]:amount end)/POW(10,6)  as amount
+FROM 
+{{ ref('terra__transactions') }}
+WHERE fee_denom IS NOT NULL 
+  --AND fee_denom in ('UST', 'SDT', 'AUT', 'CAT', 'EUT', 'JPT', 'KRT', 'LUNA', 'MNT')
```
[the remark this answers is not in the document]
Comment (amasucci13) [reply models/consoles/terra/consoles__terra_daily_total_fees_USD.sql]: I deleted this comment

File: tests/terra/consoles/consoles__terra_fees_UST__block_date-assert_no_gap.sql (hunk 1 above)
Comment (amasucci13) [reply tests/terra/consoles/consoles__terra_fees_UST__block_date-assert_no_gap.sql:2]: Yes, I think so because we don't want to see gaps for UST or Luna

**Review by juls858 — APPROVED** [review]

**Pull request merged**